subxt/examples/setup_config_signed_extension.rs

```diff
@@ -31,7 +31,10 @@ impl Config for CustomConfig {
             signed_extensions::CheckGenesis<Self>,
             signed_extensions::CheckMortality<Self>,
             signed_extensions::ChargeAssetTxPayment<Self>,
-            signed_extensions::ChargeTransactionPayment,
```

Comment:
prob worth keeping this in the default list so it'll keep working with chains that don't wrap the charge asset signed ext (we'll just ignore it if they do)